### DISCUSSION TITLE
Thème couleur

### DISCUSSION THREAD

**Discussion by adriens** [lschaeffer313/domaine-nc-mobile#62] [Polls]
# :grey_question: Contexte

Actuellement l'app a des couleurs jaunes et gris, qui ne rend pas la lecture optimale:

![image](https://user-images.githubusercontent.com/5235127/204767153-36134796-84c6-4ab6-9f56-f902b4b7e3a5.png)

:point_right: Le but de cette discussion est de choisir enembles un thème couleur

**Comment by HakumenNC** [comment]
La question est vite répondue 😄

Thème Noir ⚫  /Jaune 🟡  en PROD 🚀

**Reply by adriens** [reply]
"La question est vite répondue smile" MAIS NOOOOOON XPTDRRRRRRRRRRRRRRRRRRRRRRRRR

**Comment by adriens** [comment]
Yep, y'a pas photo. Bien stylé... on dirait une lamborghini !!!
